packages/ibm-products-styles/src/components/Datagrid/styles/addons/_CustomizeColumnsTearsheet.scss

```diff
@@ -5,8 +5,10 @@
 // LICENSE file in the root directory of this source tree.
 //
 
+// stylelint-disable carbon/layout-token-use
```

Comment:
Does namespacing cause these to get flagged? 🫠

Reply:
Yes, that was it ^^